src/forms/Textarea.jsx

```diff
@@ -113,82 +117,47 @@ export class Textarea extends React.PureComponent<Props, State> {
 			name,
 			value, // eslint-disable-line no-unused-vars
 			label,
-			labelClassName,
-			className,
-			error,
+			labelClassName, // eslint-disable-line no-unused-vars
 			rows,
 			style = {},
 			maxHeight,
 			minHeight,
 			maxLength,
 			id,
 			onChange, // eslint-disable-line no-unused-vars
-			autosize,
 			helperText,
 			required,
-			requiredText,
 			disableResize,
 			...other
 		} = this.props;
 
-		const classNames = {
-			textarea: cx(
-				'span--100',
-				{
-					'field--error': error,
-					'textarea--autoheight': autosize,
-					'textarea-no-resize': disableResize,
-				},
-				className
-			),
-			label: cx(
-				'label--field',
-				{
-					'label--required': required,
-					'flush--bottom': helperText,
-				},
-				labelClassName
-			),
-			helperText: cx('helperTextContainer', { required }),
-		};
-
 		// Character limits should be a "soft" limit.
 		// Avoid passing maxLength as an HTML attribute
 		if (maxLength) delete other.maxLength;
 
 		return (
 			<div className="inputContainer">
-				{label && (
-					<label
-						className={classNames.label}
-						htmlFor={id}
-						data-requiredtext={required && requiredText}
```

Comment:
We're no longer supporting this? What if the field is required?

Reply:
We don't have specs but it'll come up in vqa and we can address it then.

cc: @sadafie